nominees/orange.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Orange",
+  "description": "Orange is an open-source data visualization, machine learning and data mining toolkit. It features a visual programming front-end for explorative rapid qualitative data analysis and interactive data visualization.",
+  "website": "https://orangedatamining.com/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/biolab/orange3/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceURL": "https://core.ac.uk/download/pdf/231161571.pdf"
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceURL": "https://orangedatamining.com/home/interactive_data_visualization/"
+    }
+  ],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/biolab/orange3",
+  "organizations": [
+    {
+      "name": "Bioinformatics Laboratory",
```

Comment:
@ijeomaemeruwa could you provide the full form for the name field: `University of Ljubljana, Faculty of Computer and Information Science, Bioinformatics Laboratory`. Thanks.